homeassistant/components/vlc_telnet/media_player.py

```diff
@@ -238,6 +240,26 @@ def set_volume_level(self, volume):
         self._vlc.set_volume(volume * MAX_VOLUME)
         self._volume = volume
 
+    def volume_up(self):
```

Comment:
Isn't this already handled by the base media player entity?

https://github.com/home-assistant/core/blob/98be703d90e44efe43b1a17c7e5243e5097b00b1/homeassistant/components/media_player/__init__.py#L754-L765

Reply:
Oops, yes. Those work fine -- removed my implementations.